### PR TITLE
Added dumping of display information including EDID

### PR DESCRIPTION
**Pull request by JamesH65**
Tested with 1 HDMI, 2 HDMI, 1 HDMI+1 LCD, all seems to work OK.

### CODE REVIEW

**Comment by JamesH65** [comment]
@pelwell @6by9 For review.

**Comment by 6by9** [comment]
Great idea. Not sure the execution is perfect.
tvservice won't work on full KMS, and KMS is the direction we want to go (so I've been told).

tvservice -n for the name doesn't give us anything that isn't in the EDID, so is redundant.

/sys/class/drm/card1-*/edid is the better option in some ways.
/sys/class/drm/card1-*/modes gives you the list of supported modes (why no refresh rates?!)

```xrandr --verbose```? Or use modetest to dump the modes (I just wish it were in a package).

We do still need the tvservice side for those still in legacy mode, but that should be the fallback rather than default.

**Comment by JamesH65** [comment]
Is there an easy way to determine if we are in KMS /FKMS mode?

**Comment by JamesH65** [comment]
Actually I believe just checking for sys/class/drm will be sufficient

**Comment by 6by9** [comment]
Note that KMS and FKMS on Pi0-3 will have displays attached to /sys/class/drm/card0, whilst on Pi4 it will be /sys/class/drm/card1 (card0 is v3d for 3D processing only).

File: raspinfo/raspinfo [hunk 1]
```diff
@@ -73,6 +73,75 @@ echo
...
+         if [ -f $card/modes ];
+         then
+            cat $card/modes
+         else
+            echo "No modes found"
+         fi
+
+         if [ -f $card/edid ];
+         then
+            cat $card/edid | base64
```
Comment (6by9) [inline raspinfo/raspinfo]: Why not just ```base64 $card/edid```?

**Comment by 6by9** [comment]
Minor nit, otherwise I'm happy.

File: raspinfo/raspinfo (hunk 1 above)
Comment (JamesH65) [reply raspinfo/raspinfo]: Indeed... fixed.

**Pull request merged**